src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,31 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin onDesctop {
+  @media (min-width: 1024px) {
+    transition: all $transition-time;
```

Comment:
Why do you have your transition property in media-queries and with transition-property: all, that's not okay, remove it: